### DISCUSSION TITLE
How to I apply a custom style to the header of a paragraph?

### DISCUSSION THREAD

**Discussion by kangning0125** [quarto-dev/quarto-cli#5988] [Q&A] [answered]
### Description

I would like to replicate a style like this in a book project.
<img width="628" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/29698202/94af0f82-77b1-4e70-8452-45578d373860">

When I add a html class to the header `{.unnumbered .blue_header}`. The style applies to the whole block including the paragraph underneath it.
How can I only format the specific line of header without messing the rest of contents?

**Comment by mcanouil** [comment]
Use spans, see <https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans>.

```qmd
## [somethings]{.blue_header}
```

Or actually use CSS which in this case is controlled at least by `h2` elements.

**Comment by kangning0125** [comment]
> Use spans, see https://quarto.org/docs/authoring/markdown-basics.html#divs-and-spans.
> 
> ```
> ## [somethings]{.blue_header}
> ```
> 
> Or actually use CSS which in this case is controlled at least by `h2` elements.

I would like to use the 2nd option by assigning a css style to the class `blue_header`. However, when I use `## [somethings]{.blue_header}`, the whole paragraph beneath it is changed to the same style. How do I avoid that?

```
## Scope {.unnumbered .blue_header_line}

A model refers to a quantitative method, system, or approach that applies statistical, economic, financial,
or mathematical theories (that imply an abstraction of real-life phenomena), techniques, and assumptions
to process input data into quantitative outputs (estimates or predictions).

```

css
```
.blue_header {
    background-color: #023a6f;
    font-weight: bold;
    color: white;
    }
```

**Reply by mcanouil** [reply]
Please watch carefully the syntax of spans, it's not the one you used in your original post.
Note the square brackets.

**Reply by baptiste** [reply, marked as answer]
it's useful in such cases to take a look at the browser's Dev tools and check what is being selected. You want to select the h2 element from that section, not the whole section:

```
h2.blue_header {
           background-color: #023a6f;
           font-weight: bold;
           color: white;
           }
```

![Screenshot 2023-06-21 at 19 08 24](https://github.com/quarto-dev/quarto-cli/assets/18970/bb0c5262-4d96-4df3-a5a1-ea021f7b2c19)
